catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -63,7 +63,7 @@ const SearchTag = ({
         'flex--v-center': true,
         pointer: true,
         [spacing({ s: 1 }, { padding: ['left', 'right'] })]: true,
-        [spacing({ s: 1 }, { margin: ['right'] })]: true,
+        [spacing({ s: 1 }, { margin: ['left'] })]: true,
```

Comment:
This could go in the `spacing` function on the line above. I realise you're just dealing with what was already there, though.